data_science/palette/data/README.md

```diff
@@ -0,0 +1 @@
+This project uses images on [`/works`](https://wellcomecollection.org/works) (in thumbnail form), stored locally. However, adapting the notebooks to use the API itself should be trivial.
```

Comment:
minor: I’m not a fan of saying something should be “easy” or “trivial” in documentation. If it’s actually true, we don’t need to say it. If somebody finds it harder, it’s frustrating.

If it’s really trivial, let’s do it right now!

If not, just say something like “It would be possible to adapt the notebooks to use the catalogue API directly”